lakeview/src/main/java/ai/onehouse/metrics/LakeViewExtractorMetrics.java

```diff
@@ -64,6 +64,14 @@ public void incrementTableDiscoveryFailureCounter() {
     metrics.increment(TABLE_DISCOVERY_FAILURE_COUNTER, getDefaultTags());
   }
 
+
+  public void incrementTableDiscoveryFailureCounter(
+        MetricsConstants.MetadataUploadFailureReasons metadataUploadFailureReasons) {
+    List<Tag> tags = getDefaultTags();
+    tags.add(Tag.of(TABLE_DISCOVERY_FAILURE_COUNTER, metadataUploadFailureReasons.name()));
```

Comment:
looks like this metric name is already used in other parts of the code. A given metric should always have same tag combinations applied otherwise it will not be able to capture all the metrics. Make sure all the places where a given metric name is used has similar tags mentioned.

Eg: `metric_name{tag1="value1", tag2="value2"}` can't work along with `metric_name{tag1="value1", tag2="value2", tag3="value3"}`